truss/templates/server/common/truss_server.py

```diff
@@ -88,41 +119,29 @@ def is_binary(request: Request):
         )
 
 
-class TrussServer(kserve.ModelServer):
+class TrussServer:
 
     _endpoints: BasetenEndpoints
     _model: ModelWrapper
     _config: Dict
 
     def __init__(self, http_port: int, config: Dict):
-        super().__init__(
-            http_port=http_port,
-            enable_grpc=False,
-            workers=1,
-            enable_docs_url=False,
-            enable_latency_logging=False,
-        )
-
+        self.http_port = http_port
+        self.enable_grpc = False
```

Comment:
should we remove `enable_grpc`, `enable_docs_url`, `enable_latency_logging`, and `max_asyncio_workers` now that we control the implementation?

Reply:
Will do